src/utils/install/journal_defaults.json

```diff
@@ -5132,5 +5132,24 @@
         "value": {
             "default": ""
         }
+    },
+     {
+        "group": {
+            "name": "general"
+        },
+        "setting": {
+            "description": "Whether or not to use CRediT for this journal",
+            "is_translatable": false,
+            "name": "use_credit",
+            "pretty_name": "Enable CRediT",
+            "type": "boolean"
+        },
+        "value": {
+            "default": "on"
```

Comment:
Should this be on by default? It causes a state change that editors may not be aware of.

Reply:
Yeah, good point, and there are many journals for whom CRediT does not provide the right taxonomy.

I am hesitant to turn it off by default and do so silently, because that may mean lots of people don't use it who could benefit from it and would want to use it if they knew about it.

Here's a possible solution in two parts:
1) turn it off by default
2) create an interactive upgrade guide in Janeway that shows up in a prominent banner above the top of the manager, for press managers and journal managers when they log into a new version that has important decisions for them to make. They can dismiss it without opening it to accept defaults, or they can go through the items to make decisions, after which it will also go away.

Reply:
If #2 takes too long to do for 1.8, we could create it and include some important settings retroactively that may have been missed, like this one. What do you think?

Reply:
Sounds like a great idea for surfacing changes after upgrades but yes potentially too much for 1.8. Can you write this up and mark it for discussion at the next BRF?